documentation/swagger-good-patterns.md

```diff
@@ -66,4 +66,24 @@ These are patterns that a spec is expected to follow by default to be included i
  - **Specify units in Description**
  
    - It is a good practice to specify units in description for properties like size
- - **Add a link to MSDN Rest Api Docs whenever possible.**
+ 
+ - **Add a link to MSDN Rest Api Docs/examples whenever possible.**
+ 
+ - **Mutability**
+ 
+	- Use x-ms-mutability extension to document mutability aspects of models
+	- `Readonly: true` should be applied on all read-only properties
+ 
+ - **Required properties**
+ 
+	- `Required: true` should be applied on all required properties
```

Comment:
This is incorrect. One can specify a property as required by adding that property to the "required" array.
Example:
```
"Resource": {
  "properties": {
    "id": {
      "description": "Resource ID",
      "readOnly": true,
      "type": "string"
    },
    "name": {
      "description": "Resource name",
      "readOnly": true,
      "type": "string"
    },
    "location": {
      "description": "Resource location",
      "type": "string"
    }
  },
  "required": [ "location"]
}
```